extensions/vertx-http/deployment/src/main/java/io/quarkus/devui/deployment/BuildTimeContentProcessor.java

```diff
@@ -245,7 +262,7 @@ void gatherMvnpmJars(BuildProducer<MvnpmBuildItem> mvnpmProducer, CurateOutcomeB
         ClassLoader tccl = Thread.currentThread().getContextClassLoader();
         try {
             Enumeration<URL> jarsWithImportMaps = tccl.getResources(Location.IMPORTMAP_PATH);
-            Set<URL> jarUrls = new HashSet<URL>(Collections.list(jarsWithImportMaps));
+            Set<URL> jarUrls = new HashSet<>(Collections.list(jarsWithImportMaps));
```

Comment:
Is there a reason it is a set? Wouldn't `Collections.list(jarsWithImportMaps)` be enough? Or even iterating through the original enumeration?

Reply:
True, changed to just enumerate the original enumeration